### PR TITLE
WIP: adds new site

### PR DESCRIPTION
**Pull request by zonzujiro**
Also, we need to add SSH keys for github action - https://github.com/peaceiris/actions-gh-pages#getting-started

@suchov pls provide @AMashoshyna with permissions. I don't have them

### CODE REVIEW

**Comment by AMashoshyna** [comment]
@zonzujiro Please add `.babelrc` and `.eslintrc` files, otherwise build crashes

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-kottans.org
```
Comment (suchov) [inline CNAME]: hmmm, not sure why are you deleting this?
it is a part of the domain policy at github to use a custom domain instead of .github.io

Comment (suchov) [reply CNAME]: https://medium.com/@hossainkhan/using-custom-domain-for-github-pages-86b303d3918a - as an example of how to config it

Comment (zonzujiro) [reply CNAME]: Done. 
Thank you!

**Comment by zonzujiro** [comment]
@AMashoshyna @suchov ok, I will ask one more time - CAN I MERGE!? 😆

**Comment by AMashoshyna** [comment]
force-pushing branches? That's mean =)
I'm on it, give me some more time pls

**Comment by AMashoshyna** [comment]
LGTM

**Pull request merged**